### ISSUE TITLE
Native Modules can't be required for second time in Web Workers

### ISSUE THREAD

**Issue by gpetrov** [nwjs/nw.js#7239]
NWJS Version : 0.42.5 - 0.37.4
Operating System : Windows 10

### Expected behavior

After terminating Web Worker, on a new worker creation, the require of native module should work fine

### Actual behavior

Require of the native module works - but only once. After the worker is terminated, on a new start the require gets "Module did not self-register." error

### How to reproduce

Run the included test and you will see in the console that the in first start of the worker, the require works fine, but not on the next start.

![error_loading_module](https://user-images.githubusercontent.com/504730/69497546-89769500-0ede-11ea-9864-a6edb3f4be37.png)

[webworker_native_module.zip](https://github.com/nwjs/nw.js/files/3883800/webworker_native_module.zip)

The sample is with sqlite3 but the problems is with any native module.

**Comment by gpetrov** [comment]
@rogerwang is this maybe related to Worker Threads?

I also tried to add --experimental-worke to the js-flags, but it didn't made any difference

https://groups.google.com/forum/#!topic/nwjs-general/QpO31WA4fIA

Also seems native modules have general difficulty in being unloaded and loaded again.
We tried to implement hot reloading of node modules (by just removing them from the require.cache) but hit a wall with native modules in NWJS - as those weren't able to reload.
